aggsender/db/aggsender_db_storage.go

```diff
@@ -179,10 +211,38 @@ func getCertificateByHeight(db dbtypes.Querier,
 		"SELECT * FROM certificate_info WHERE height = $1;", height); err != nil {
 		return nil, getSelectQueryError(height, err)
 	}
-
 	return &certificateInfo, nil
 }
 
+// getCertificatesByHeight returns a certificates in the required table
+func getCertificatesByHeight(db dbtypes.Querier, table tableName,
+	height uint64) ([]*certificateInfo, error) {
+	var certificates []*certificateInfo
+	if err := meddler.QueryAll(db, &certificates,
+		fmt.Sprintf("SELECT * FROM %s WHERE height = $1;", table), height); err != nil {
+		return nil, err
+	}
+	return certificates, nil
+}
+
+// getCertificatesHeightOlderThanHeight returns a list of certificate heights older than the provided height
+func getCertificatesHeightOlderThanHeight(db dbtypes.Querier, table tableName,
```

Comment:
Consider making this a method, so then you can remove the db param and reuse the db reference from the instance. WDYT?
```suggestion
func (a *AggSenderSQLStorage) getCertificatesHeightOlderThanHeight(table tableName,
```

Reply:
Also maybe it doesn't make sense parameterizing the `tableName`, as it is always the `certificate_info` at the moment. WDYT?

Reply:
> Consider making this a method, so then you can remove the db param and reuse the db reference from the instance. WDYT?
Same response than before: https://github.com/agglayer/aggkit/pull/1057#discussion_r2393754660
>Also maybe it doesn't make sense parameterizing the tableName, as it is always the certificate_info at the moment. WDYT?
In fact, with the refactor sugestion here https://github.com/agglayer/aggkit/pull/1057#discussion_r2391220489 I would need to call also in history table to get the filenames of the certificates to delete